report/3.Theory/LanguageSpecification.tex

```diff
@@ -47,17 +49,17 @@ \subsection{Types}
 
 \subsection{Operators}
 Most of the operators in HCL are defined as functions.
-This means that all arithmetic operators are infix-functions that take two numbers as parameters. 
-The only operators that remain operators are the assign-operator (\textbf{=}) and the encapsulation-operators, namely parenthesis, square-brackets and curly-brackets.
+This means that all arithmetic operators are infix-functions, which has expressions as parameters. 
+The only operators that remain operators are the assign-operator and the encapsulation-operators, namely parenthesis, square-brackets and curly-brackets.
 
-Not having operators for arithmetic operations also means that there are no native operator precedence in HCL.
-This can be amended by structuring calculations in a left-to-right manner, or by encapsulating operations with parentheses.
+Not having operators for arithmetic operations also means that there is no native operator precedence in HCL.
+This can be amended by structuring calculations in a left-to-right manner, or by encapsulating operations with parentheses, since parentheses has higher precedence than operators.
 
 \subsection{Functions}
 \textbf{Function Declaration}\\
 The type of a function's parameters, as well as its return type, are explicitly declared when a function is defined. 
 The declaration is structured as an assignment, where the functions name is declared on the left-hand-side, prefixed by either \textbf{\textit{var}} or \textbf{\textit{func}}. 
-The parameters of the function is declared on the right-hand-side inside parentheses, followed by a colon (\textbf{\textit{:}}), which declares the return type. 
+The parameters of the function is declared on the right-hand-side, inside parentheses, followed by a colon, which declares the return type. 
```

Comment:
"The parameters of the function is"
skal ændres til
"The parameters of the function are"